modules/tinymce/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tinymce",
-  "version": "8.0.1",
+  "version": "8.0.2",
```

Comment:
We shouldn't need to do this during regular PRs, it should be bumped during the release process. This implies the release was not done correctly.

- [ ] Revert and ping @jscasca about the issue

Reply:
this was carried over during branching, I got rid of it